b2b-sample/README.md

```diff
@@ -186,6 +188,8 @@ setup the config json file as mentioned above.
  # From `<SAMPE_IS_HOME>/b2b-sample`
  nx serve business-app
 ```
+> **_NOTE:_** If `nx serve business-app` produced an error, try with `npx nx serve business-app`
```

Comment:
Suggestion: `If nx serve business-admin-app produces an error, try using npx nx serve business-admin-app.`

Check the above one as well

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c4229e64c82f57a16a579a24259eafab0e6cc56f